README.md

```diff
@@ -65,25 +64,29 @@ The Helix CLI tool can be used to check, compile and deploy Helix locally.
 3. Write queries
 
    Open your newly created `.hx` files and start writing your schema and queries.
-   Head over to [our docs](https://docs.helix-db.com/introduction/cookbook/basic) for more information about writing queries
+   Head over to [our docs](https://docs.helix-db.com/documentation/hql/hql) for more information about writing queries.
+
+
 
    ```js
-   QUERY addUser(name: String, age: I64) =>
-      user <- AddN<User>({name: name, age: age})
+   N::User {
+      name: String,
+      age: U32
+   }
       RETURN user
```

Comment:
There's a random RETURN here. Also maybe add a comment what "name" should be INDEXed for good performance?